.travis.yml

```
@@ -2,5 +2,4 @@ language: ruby
 rvm:
   - 2.6.3
   - 2.5.5
```

Should we switch this to the supported versions of Ruby? I'm hoping `3.0.2` doesn't break. :)

```suggestion
  - 3.0.2
  - 2.7.4
  - 2.6.8
```